examples/typescript/package.json

```diff
@@ -30,6 +32,6 @@
     "schema":
       "apollo-codegen introspect-schema https://mpjk0plp9.lp.gql.zone/graphql --output ./src/schema.json",
     "types":
-      "apollo-codegen generate --schema schema.json --target typescript-modern `find ./src -name *.graphql`"
+      "apollo-codegen generate ./src/**/queries.ts --addTypename --schema ./src/schema.json --target typescript --output ./src/__generated__/types.ts"
```

Comment:
We'd rather be using `typescript-modern` here - as `typescript` is going to be deprecated and all efforts are in the `modern` variants.  I know it doesn't say that anywhere but I discovered it in chat.

Reply:
it is brocker right now, bacause of a dependency on a beta version on babel types

Reply:
Ah, got it.  Yeah I'm on a private release.